packages/website/plugins/generated-rule-docs.ts

```diff
@@ -193,35 +194,79 @@ export const generatedRuleDocs: Plugin = () => {
         type: 'paragraph',
       } as mdast.Paragraph);
 
+      // need this, so we don't have comments inside the JSON objects
+      const getEslintrcStr = (withComment: boolean): string => {
+        return `{
+  "rules": {${
+    withComment
+      ? '\n    // Note: you must disable the base rule as it can report incorrect errors'
+      : ''
+  }
+    "${extendsBaseRuleName}": "off",
+    "@typescript-eslint/${file.stem}": "${optionLevel}"
+  }
+}`;
+      };
+
       root.children.splice(howToUseH2Index + 1, 0, {
         lang: 'js',
         type: 'code',
         meta: 'title=".eslintrc.cjs"',
-        value: `module.exports = {
-  "rules": {
-    // Note: you must disable the base rule as it can report incorrect errors
-    "${extendsBaseRuleName}": "off",
-    "@typescript-eslint/${file.stem}": "${optionLevel}"
-  }
-};`,
+        value: `module.exports = ${getEslintrcStr(true)};`,
       } as mdast.Code);
+
+      root.children.splice(howToUseH2Index + 2, 0, {
+        children: [
+          {
+            type: 'link',
+            url: `/play#eslintrc=${convertToPlaygroundHash(
+              getEslintrcStr(false),
+            )}`,
+            children: [
+              {
+                type: 'text',
+                value: 'Click here to try this rule in the playground',
```

Comment:
Nit: not everybody "clicks". Some tap, or press enter, or use other things. We can just remove that bit of the text.

```suggestion
                value: 'Try this rule in the playground ↗',
```